packages/front-end/components/Features/ConditionDisplay/index.tsx

```diff
@@ -216,7 +249,7 @@ function getConditionParts({
           {operatorToText(operator, renderPrerequisite)}
         </span>
         {hasMultiValues(operator) ? (
-          <MultiValueDisplay value={value} />
+          <MultiValueDisplay value={value} displayMap={displayMap} />
```

Comment:
I think this might break for actual `$inGroup` and `$notInGroup` operators (list saved groups)